msteams-platform/m365-apps/actions-in-m365.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Actions in Microsoft 365
+description: In this article, learn more about the function of Actions and its use cases. 
+ms.date: 11/16/2023
+ms.author: mosdevdocs
+author: Vishnu
+ms.topic: Conceptual
+ms.subservice: m365apps
+---
+# Actions in Microsoft 365
+
+> [!NOTE]
+>
+> * Actions for Microsoft 365 is available in [public developer preview](../resources/dev-preview/developer-preview-intro.md).
+>
+> * Actions is supported for Microsoft 365 apps on web and desktop clients only and isn't supported on Outlook and Microsoft Teams.
+
+Actions enable your app to integrate seamlessly into the user's workflow and guide users to your app based on their intent and the content at hand, making task completion more efficient.
+Actions help you enhance your app's visibility and engagement with minimal development effort and improve user productivity by streamlining task completion and reducing the need to switch contexts across various Microsoft 365 apps. For instance, your app can act immediately on content files, expanding the range of interactions users can have with their content.
+
+You can extend your Teams app across Microsoft 365 using Actions. However, it's important to note that Actions function only in Microsoft 365.
+
+In the following graphic, the user intends to attach a Word document to the list in the **To do** app. The user right-click on the Word document and select the **Add to** action. A dialog appears with the Word document attached. The user adds a note and selects **Submit**. The Word document is added to the list in the **To do** app:
+
+  :::image type="content" source="images/actions-in-m365-app.gif" alt-text="Graphic shows the user can add a file to the to-do list app with a note attached for a task to complete.":::
+
+## Build Actions
+
+Actions are the combination of intent, object, and handler. Actions represent the task that the user wants to perform where intent is the user’s desired action, object is the function to be executed, and handler is the way to perform the action on the object.
+
+When you're creating an app ensure that you define user intent, choose the object to perform the action, and construct the corresponding handler that facilitates task completion for the user.
```

Comment:
This description is a big vague. Could we use similar text to dev preview documentation:

To build an action, define the intent, object, and handler of the action in your app manifest. In your handler code, use the TeamsJS library to receive the action information and perform your user's requested task on the specified file.  